src/antennachecker/test/check_antenna_test.tcl

```diff
@@ -0,0 +1,9 @@
+
+# read the skywater LEF
+read_lef /afs/eecs.umich.edu/vlsida/projects/openroad/users/wenbo/AC_unpadded/merged_unpadded.lef
+
+# read the striVe_soc DEF
+read_def -order_wires /afs/eecs.umich.edu/vlsida/projects/openroad/users/wenbo/AC_unpadded/Antenna_Checking/striVe_soc_s2/striVe_soc.def
```

Comment:
no absolute pathnames allowd anywhere. this will not work on a gcloud server!
shows the tests are not running on the CI. see openroad/regression script to add them.

Reply:
Yes, sorry about that. We got the approval to use skywater DEF/LEF now. So this is fixed now.